### DISCUSSION TITLE
Debug messages for notifications?

### DISCUSSION THREAD

**Discussion by hwwilliams** [containrrr/watchtower#744] [Q&A] [answered]
Hi,

I normally use AWS SES for my SMTP server on self-hosted applications and I've usually had very little trouble getting it to work but I can't get Watchtower to send notification emails no matter what ports I use (25, 465, or 587).

I don't know what's wrong because as far as I can tell there are no messages related to notifications in the docker log output when Watchtower debugging is enabled.

Are there logs for the notifications?

Thank you,

**Comment by simonaronsson** [comment, marked as answer]
Hey!

You should be able to set the env var `SHOUTRRR_DEBUG` to `true` to have shoutrrr output debug logs

**Reply by hwwilliams** [reply]
I'm not using Shoutrrr, I'm just using the built-in email service with the AWS SES SMTP server information.

**Reply by simonaronsson** [reply]
Sorry, my bad. We're looking to remap all the legacy notification services to shoutrrr and my memory (falsely) had me believe we'd already merged that change.

In any case, I would suggest that you switch to using shoutrrr email notification as a first step, since #450 (or even a break-out PR from that one) will remove most of the current notification code altogether.

**Reply by hwwilliams** [reply]
Is Shoutrrr included in the Watchtower container?

I don't see any installation instructions for Shoutrrr in the [documentation page](https://containrrr.dev/shoutrrr/getting-started/).

**Reply by simonaronsson** [reply]
It is! See https://containrrr.dev/watchtower/notifications/#containrrrshoutrrr

**Reply by hwwilliams** [reply]
Well at least now I see an error message. I switched to using Shoutrrr for email, but it says connection refused no matter what port I use, does the number '25' refer to a line or the port its trying to connect to?

`Failed to send notification email:  dial tcp :25: connect: connection refused`

**Reply by piksel** [reply]
it does, but there's usually a hostname preceding the port (and if you don't specify any port it defaults to 25)
